XMLParser/XMLParser.cpp

```diff
@@ -26,7 +26,7 @@ bool XMLParser::ParseFile(const std::string& filename)
 	std::ifstream file{ filename };
 	if (!file.is_open())
 	{
-		LOG_INFO << "Can't open file.";
+		//LOG_INFO << "Can't open file.";
```

Comment:
remove commented code

Reply:
This is only for demo
Logger in new branch is using separate parser